### DISCUSSION TITLE
🎉 Manifest 4.1.1 "Clank" is here!

### DISCUSSION THREAD

**Discussion by SebConejo** [mnfst/llm-gateway#240] [Announcements]
We’re excited to announce the release of Manifest 4.1.1, bringing notable improvements:

- ☝️ **Single types**: Easily edit singular entities like dynamic pages, settings, or layout content.
- ✍️ **Rich text input**: Enjoy a rich text editor supporting links, headings, and formatting.

These features open the door for developers to create new types of projects with Manifest, such as **CMSs**. 

Check out the full release notes here: [Manifest 4.1.1 Release Notes](https://github.com/mnfst/manifest/releases/tag/4.1.1).

Thank you for your continued support! 💪 We can’t wait to see what you build! 🚀

**Comment by brunobuddy** [comment]
Closing this out. It's from Manifest's old backend-as-a-service days, which we don't work on anymore since the pivot. See #742 for where things are now.
